src/iuse.cpp

```diff
@@ -8445,8 +8445,14 @@ int iuse::camera(player *p, item *it, bool, point)
         }
 
         if (aim_point.x == p->posx() && aim_point.y == p->posy()) {
-            p->add_msg_if_player(_("You decide not to flash yourself."));
-            return 0;
+            if (query_yn(_("Are you sure you want to flash yourself?"))) {
+                p->add_msg_if_player(_("Great selfie. The flash blinds you."));
+                p->add_effect("blind", rng(5, 10));
```

Comment:
But it's sunny out (no need for flash!)/I've got sunglasses on/I've got antiglare gear on/etc!

More to the point, I can understand the use of a short-term self-blinder, but frankly we'd be better off with a blindfold (if one hasn't yet been made) rather than making people pull nethack stunts.  :-/  Sorry.